app/assets/javascripts/application.js

```diff
@@ -14,4 +14,21 @@
     GOVUK.modules.start();
 
   });
+
+  $(function() {
+    $('.related-topic-links').each(function(i) {
+      if (i > 0) {
+        var $this = $(this);
+        var title = $this.find('h2').text();
+        $this.find('ul').hide();
+        $this.find('.reveal').text('See related links in '+title).on('click', function(e) {
+          e.preventDefault();
+          $this.find('ul').show();
+          $(this).text('See everything in '+title).on('click', function(e) {
+            window.location = this.href;
```

Comment:
As mentioned IRL, it could be a bit weird for this to act like a button, then mutate to act like a link (buttons do things, but don't go anywhere; links _always_ go places and might sometimes also have behaviour)